### ISSUE TITLE
New version with updated rest-client dependency

### ISSUE THREAD

**Issue by 97jaz** [auth0/ruby-auth0#93]
We've run into the problem described in #83, and it appears that a fix was committed in April, which consists solely of updating the version of `rest-client` (#87). Do you have plans to release a new version of the `auth0` gem soon?

**Comment by nonnenmacher** [comment]
Got into the same trap, updating (on MacOS) from ruby 2.4.0 to 2.4.1 because some deprecations (OpenSSL:Cipher::Cipher) that are now enforced in the implied brew update of OpenSSL librairie.

I changed my Gemfile to read the latest gemspec dependency on rest-client -> '2.0'
with 
```
gem  'rest-client', "~> 2.0"
gem 'auth0', '~> 4.1' , github: 'auth0/ruby-auth0'
```

and everything went smoothly.

**Comment by wingrunr21** [comment]
@lobo Can a new release be cut? Last rubygems release was July 2016

**Comment by aaguiarz** [comment]
Hi

We apologize it, we'll take care of it soon.

Regards,

Andres

**Comment by bramski** [comment]
Poke.  Please follow through @aaguiarz.  I'm waiting on this as well.

**Comment by jqr** [comment]
@bramski looks like this was released in [v4.2.0](https://github.com/auth0/ruby-auth0/blob/master/CHANGELOG.md#v420-2018-02-15) yesterday.

**Comment by aaguiarz** [comment]
We just published v4.4.0 to Rubygems

**Comment by bramski** [comment]
thanks guys!
